addons/test_mass_mailing/tests/test_mailing_test.py

```diff
@@ -65,6 +65,7 @@ def test_mailing_test_equals_reality(self):
         """
         Check that both test and real emails will format the qweb and inline placeholders correctly in body and subject.
         """
+        self.env.ref('mass_mailing.mass_mailing_contact_0').name = 'Mitchell Admin'
```

Comment:
Mmmh ... Weird. As the mailing test tool takes "the first found record of the mailing domain", maybe we could unlink all mailing contacts in the test (should not have much anyway) so that we effectively rely on the contact created in the contact list.